Package.swift

```diff
@@ -8,13 +8,6 @@ let package = Package(
     .plugin(name: "FormatSwift", targets: ["FormatSwift"]),
   ],
   dependencies: [
-    .package(url: "https://github.com/calda/SwiftFormat", exact: "0.49.11-beta-2"),
-    // The `SwiftLintFramework` target uses "unsafe build flags" so Xcode doesn't
-    // allow us to reference a specific version number. To work around that,
-    // we can reference the specific commit for that version.
```

Comment:
This was actually also blocking consumers from specifying version numbers like `from: 1.0.0`, so that was another reason why we needed to move to prebuilt binary executables.